scripts/install.sh

```
@@ -918,12 +921,34 @@ check_veth_menu() {
   return 1
 }
 
+ensure_dns() {
+  local count
+  count="$(trim "$(ros_cmd ':local d ""; :do { :set d [/ip/dns/get dynamic-servers] } on-error={}; :put ([:len [/ip/dns/get servers]] + [:len $d])' 2>/dev/null || true)")"
```

Shellcheck (0.11.0)

[info] Expressions don't expand in single quotes, use double quotes for that.

(SC2016)